src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -127,8 +111,9 @@ public IDAMProperties getUserDetails() {
      * @param userId user id
      * @return UserDefault
      */
-    private UserDefault getUserDefaultSettings(final String userId) {
+    public UserDefault getUserDefaultSettings(final String userId) {
         try {
+            LOG.debug("retrieving default user settings");
             final HttpEntity requestEntity = new HttpEntity(securityUtils.authorizationHeaders());
```

Comment:
Just wondering whether you would like to debug also userId

LOG.debug("retrieving default user settings for userId {}", userId);

Reply:
ok sure